src/OpenTelemetry.AutoInstrumentation.StartupHook/StartupHook.cs

```diff
@@ -69,28 +69,25 @@ public static void Initialize()
 
             if (profilerType != null)
             {
-                Logger.Error(
-                    $"OpenTelemetry.AutoInstrumentation assembly was already loaded from: {profilerType.Assembly?.Location}. " +
-                    "Check the DOTNET_STARTUP_HOOK environment variable.");
+                Logger.Information(
```

Comment:
Not related to this PR, need to validate if `OpenTelemetry.AutoInstrumentation` can be loaded from native profiler without the help of StartupHook.

Reply:
Before we had the StartupHook, `OpenTelemetry.AutoInstrumentation` was loaded via the Loader the same way that is still loaded in the .NET Framework. When the StartupHook was added we opted to require its usage instead of using the Loader: so it is always the same path injecting `OpenTelemetry.AutoInstrumentation` into a .NET application. Currently, the CLR profiler will log an error and won't attach if the StartupHook is not configured - notice that the CLR Profiler `Initialize` is called before the StartupHook is loaded. /cc @nrcventura

Reply:
I added this note to remove this condition as native profiler does not load `OpenTelemetry.AutoInstrumentation` any more.